config/config.go

```diff
@@ -260,6 +260,7 @@ func parseAndValidateConfigBytes(yamlBytes []byte) (config *Config, err error) {
 			return nil, err
 		}
 	}
+	config.UI.MaximumNumberOfResults = config.Storage.MaximumNumberOfResults
```

Comment:
I'm not sure about this part, but I think that it's a bad idea to have separate configuration for MaximumNumberOfResults in UI